guide/interactions/slash-commands.md

```diff
@@ -197,11 +196,11 @@ client.on('interactionCreate', interaction => {
 });
 ```
 
-However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `CommandInteraction#isCommand()` method:
+However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `CommandInteraction#isChatInputCommand()` method:
```

Comment:
This was wrong in the first place.

```suggestion
However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `Interaction#isChatInputCommand()` method:
```

In addition to this, there is still text below this that uses `CommandInteraction`.